cloudca/resource_cloudca_vpn_user_test.go

```diff
@@ -0,0 +1,109 @@
+package cloudca
+
+import (
+	"fmt"
+	"testing"
+
+	cca "github.com/cloud-ca/go-cloudca"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/acctest"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/resource"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/terraform"
+)
+
+func TestAccRemoteAccessVPNUserCreate(t *testing.T) {
+	/*
+	 this tests conflicts with the VPN Enable test, since it
+	 expects the VPN to be disabled,
```

Comment:
make this comment the same as the one in the other test (or change both of them to something else i just think it would be nice if they were the same)

+ maybe remove the // t.Parallel() line

Reply:
d5b60431492ddeaac993ad92a69950618377e48b